controllers/datamovement_controller.go

```diff
@@ -51,8 +56,28 @@ const (
 	finalizer = "dm.cray.hpe.com"
 
 	InitiatorLabel = "dm.cray.hpe.com/initiator"
+
+	// DM ConfigMap Info
+	configMapName      = "nnf-dm-config"
+	configMapNamespace = corev1.NamespaceDefault
+
+	// DM ConfigMap Data Keys
+	configMapKeyCmd             = "dmCommand"
+	configMapKeyProgInterval    = "dmProgressInterval"
+	configMapKeyDcpProgInterval = "dcpProgressInterval"
+	configMapKeyNumProcesses    = "dmNumProcesses"
+
+	// DM ConfigMap Default Values
+	configMapDefaultCmd             = ""
+	configMapDefaultProgInterval    = 5 * time.Second
+	configMapDefaultDcpProgInterval = 1 * time.Second
+	configMapDefaultNumProcess      = 1
 )
 
+// Regex to scrape the progress output of the `dcp` command. Example output:
+// Copied 1.000 GiB (10%) in 1.001 secs (4.174 GiB/s) 9 secs left ..."
+var progressRe = regexp.MustCompile(`Copied\s\d+\.\d+\s\S{3}\s\((\d{2,3})%\)\sin.+\n`)
```

Comment:
This seems _really_ specific. Is there a reason to be so connected to the dcp output, instead of something simple? (I also think ascii character classes are more readable)
https://play.golang.com/p/nh2Xxf6gz2G

Reply:
`[[:digit:]]{1,3}%` would be more accurate than my playground example

Reply:
Really just to ensure that we're grabbing the right line, but you're right that it's overkill. I'll simplify.